src/flows/pipes/RawFluxEditor/FunctionsList/perFunction/FluxDocsTooltipContent.tsx

```diff
@@ -7,11 +7,16 @@ import {DapperScrollbars} from '@influxdata/clockface'
 // Types
 import {Fluxdocs} from 'src/client/fluxdocsdRoutes'
 
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+
 interface TooltipProps {
   item: Fluxdocs
 }
 
 const FluxDocsTooltipContent: FC<TooltipProps> = ({item: func}) => {
+  event('Flux function tooltip overlay opened')
```

Comment:
@TCL735 - I added the event metrics inside the tooltip component. so basically, whenever user hovers over the lists of flux functions, aka the tooltip component renders, it records the event. Let me know if there is a better way to do this.

Reply:
The event should be fired with some information about which function the user is looking at.

Reply:
The number of times the even should be fired is a good question. A user may hover over the list back and forth several times. Let's get feedback and guidance on how many times the event for each function should be fired.

Reply:
yeah, i should add the flux package or name to the event.l good call